src/fides/config/config_proxy.py

```diff
@@ -22,23 +22,32 @@ class ConfigProxyBase:
     Config proxy classes allow these "resolved" properties to be looked up
     as if they were a "normal" pydantic config object, i.e. with dot notation,
     e.g. `ConfigProxy(db).notifications.notification_service_type`
+
+    To implement _merge_ behavior in config value resolution for a given property,
+    the property name should be specified as an element in the `merge_properties`
+    constructor arg, which takes a set of strings.
```

Comment:
Sorry but I don't understand this at all 🙃 

Maybe an example usage?

Reply:
improved this (i think!) in https://github.com/ethyca/fides/pull/4584/commits/bfceb0364aa44ecda56adf4dd2c61282d2ad3a76